packages/block-library/src/cover/edit.native.js

```diff
@@ -364,6 +365,21 @@ const Cover = ( {
 		} );
 	}, [] );
 
+	const selectedColorText = getStylesFromColorScheme(
+		styles.selectedColorText,
+		styles.selectedColorTextDark
+	);
+
+	function getBottomLabelText() {
```

Comment:
I was wondering if we could have a variable instead of a function, what do you think?